lib/marshal/marshal.h

```diff
@@ -1,28 +1,9 @@
 #ifndef __MARSHAL_H__
 #define __MARSHAL_H__
 
-#include <stdlib.h>
-
-/* Defines for returned status */
-#define STORED          0
-#define NOT_STORED      1
-#define EXISTS          2
-#define NOT_FOUND       3
+#include "../shared.h"
```

Comment:
Sorry for the modifying your code at marshal.h, as it showed up as a redefined compiling error with shared.h, we mistakenly changed marshal.h, instead of deleting shared.h, and we will include marshal.h later. please stash, and sorry for bothering